src/winforms/toga_winforms/libs.py

```diff
@@ -31,7 +31,9 @@
 )  # noqa: E402
 
 user32 = ctypes.windll.user32
+shcore = ctypes.windll.shcore
 win_version = Environment.OSVersion.Version.Major
+win_build = Environment.OSVersion.Version.Build
```

Comment:
This is a really small thing, but could we change `win_version` to be `Environment.OSVersion.Version`, and then get access to all the structure?

Reply:
Went ahead and changed it to that.